connector/signaltometricsconnector/README.md

```diff
@@ -91,6 +92,31 @@ sum:
   [OTTL converters](https://pkg.go.dev/github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl/ottlfuncs#readme-converters)
   can be used to transform the data.
 
+#### Gauge
+
+Gauge metrics are supported for logs and allow extracting numeric values from log bodies using Grok patterns via the `ExtractGrokPatterns` OTTL function. The gauges are aggregated by last value. 
```

Comment:
Are we supporting only logs or all signal types? I was thinking that we would add support for gauges for all signal types.

Reply:
I only added support for logs, will look at other signal types also.

Reply:
The aggregator instance is shared and called for all signal types, so, I think the code should already be working for all signal types.